mozilla/kuma
# Fixes #5828, use react header on new signup flow

This changes the signup page to use the new header. I opted to create a separate page type because I see this page adopting the new frontend tech much sooner than the landing page.

## Testing

The simplest way is to:

1. Open up the landing page in an incognito page
2. Click on the sign in link
3. Now, here you might get derailed because you have already "signed up" with your Github user.
4. This is where you would then use the test account I refered to in the email sent earlier.
5. Using this account to sign-in, should bring you to the new user signup view
6. Here you want to be sure that the new frontend header is being used as apposed to the old frontend header


## Current

![Screenshot 2019-09-16 at 21 12 12](https://user-images.githubusercontent.com/10350960/64986204-cd3d9480-d8c6-11e9-9e82-62492fb7d9e7.png)

## New

![Screenshot 2019-09-16 at 21 11 09](https://user-images.githubusercontent.com/10350960/64986214-d3cc0c00-d8c6-11e9-9120-628b6fd0bc71.png)
